third-party/realdds/doc/discovery.md

```diff
@@ -75,6 +75,23 @@ The message format is a [flexible message](../include/realdds/topics/flexible/)
 All are optional except `name` and `topic-root`. Any fields not shown above will be ignored.
 
 
+# Disconnection
+
+Under normal operation, the DDS subsystem will notify participants of entity disconnections without much delay. If a participant crashes or goes offline unexpectedly, a timeout (currently 10 seconds) is triggered and only then participants are notified.
+
+When it is expected that a server will go offline, the server can elect to send a `stopping` message so that clients can remove the device immediately rather than waiting for DDS to do its thing.
+
+```JSON
+{
```

Comment:
This will be sent on the `device-info` topic? No need for the other fields?

Reply:
No, no need for the other fields